packages/dx-chart-core/src/utils/event-tracker.js

```diff
@@ -9,7 +9,16 @@ const getEventCoords = (e) => {
   ];
 };
 
-const compare = (t1, t2) => t1.distance - t2.distance;
+const DISTANCE_THRESHOLD = 20;
+
+const compare = (t1, t2) => {
```

Comment:
I think `compare` is a very common name. Very difficult to search where it used.

Reply:
More specific name is applied.